CHANGELOG.md

```diff
@@ -1,10 +1,20 @@
-# Change Log
+​# Change Log
+
+​## 1.3.20
```

Comment:
Interestingly, there are some blank characters in front of the hashes, which then break the markdown 🤔 
<img width="1161" alt="image" src="https://github.com/integromat/vscode-apps-sdk/assets/22004639/e91751e3-ecd5-41d2-8c1e-b27955c6f3b3">

```suggestion
# Change Log

## 1.3.20
```